md/templates/md.html

```diff
@@ -23,7 +23,7 @@
   </p>
 
   <p class="agency-intro">
-    <a href="{% url "md:agency-list" %}">Click here</a> to browse a list of
+    <a style="font-weight: bold; text-decoration: underline" href="{% url "md:agency-list" %}">Click here</a> to browse a list of
```

Comment:
Please use Less/CSS instead of inline styles—you could add a rule for `.agency-intro > a`, if you don't want to add a new class.